### PR TITLE
nxos: use the nxos-9 nodeset

### PR DESCRIPTION
**Pull request by goneri**

### CODE REVIEW

File: playbooks/ansible-network-nxos-appliance/files/bootstrap.yaml [hunk 1]
```diff
@@ -4,8 +4,8 @@
 - hosts: appliance
   gather_facts: false
   vars:
-    ansible_password: admin
     ansible_ssh_user: admin
+    ansible_password: qXq6vQPmU28U
```
Comment (NilashishC) [inline playbooks/ansible-network-nxos-appliance/files/bootstrap.yaml:8]: Can you use `ansible_ssh_pass` instead? I vaguely remember hitting an issue where _password doesn't work.

Comment (goneri) [reply playbooks/ansible-network-nxos-appliance/files/bootstrap.yaml]: Done

**Review by NilashishC — CHANGES_REQUESTED** [review]

File: roles/ansible-test-inventory/templates/inventory-nxos.j2 [hunk 2]
```diff
@@ -20,7 +22,7 @@ imagetag="I5"
 image_version="7.0(3)I5(1)"
```
[the remark this answers is not in the document]
Comment (NilashishC) [reply roles/ansible-test-inventory/templates/inventory-nxos.j2:22]: Could we please update this^ too?

[the remark this answers is not in the document]
Comment (goneri) [reply roles/ansible-test-inventory/templates/inventory-nxos.j2]: Done, but he test jobs don't use the file anyway.

**Pull request merged**